code/script.js

```diff
@@ -218,74 +257,196 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Ylva - Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
+//console.log(`secret person:`, secret.name);
 
-// This function to start (and restart) the game
-const start = () => {
+// Ylva - This function to start (and restart) the game - this also creates an array called charactersInPlay to be iterated
+//through later, generates the cards of the characters and sets the secret character.
+// This function also clears the RightOrWrongSection after winning or losing the "Guess who? in the game
+function start() {
+  //This shows the first option as selected in the dropdown-menu
+  charactersInPlay = CHARACTERS;
+  generateBoard();
+  setSecret();
+  console.log("The secret person is", secret.name); // - the secret person works - YEY!
```

Comment:
When you are done with the game, remember to remove this line. I forgot about it and everyone who plays my game and has the console open will see directly who the secret person is, lol.

Reply:
@edv-rd Hahaha yeah! Good thought... Will do that - once it all works :D